packages/common/src/discussions/utils.ts

```diff
@@ -1,16 +1,36 @@
 import { IGroup, IItem } from "@esri/arcgis-rest-types";
 import { IHubContent, IHubItemEntity } from "../core";
 import { CANNOT_DISCUSS } from "./constants";
+import { IRequestOptions } from "@esri/arcgis-rest-request";
+import { updateItem, updateGroup } from "@esri/arcgis-rest-portal";
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
 
 /**
  * Utility to determine if a given IGroup, IItem, IHubContent, or IHubItemEntity
  * is discussable.
- * @export
- * @param {IGroup|IItem|IHubContent|IHubItemEntity} The subject to evaluate
+ * @param {IGroup|IItem|IHubContent|IHubItemEntity} subject
  * @return {boolean}
  */
 export function isDiscussable(
-  subject: IGroup | IItem | IHubContent | IHubItemEntity
+  subject: Partial<IGroup | IItem | IHubContent | IHubItemEntity>
```

Comment:
I _think_ this is OK, i.e. I don't _believe_ this should require a breaking change.

Reply:
agreed - type "widening" should not break anything, where as type "tightening" should be a breaking change as it could cause builds to fail.